index.ts

```diff
@@ -1,12 +0,0 @@
-export * from "./client.js";
```

Comment:
Why are we remove this? This will break imports that are otherwise working today, no? Or is this a result of compiling everything into a single js and d.ts file?

Reply:
Oh this file was moved, and git didn't track it. https://github.com/cortexclick/cortex-sdk/pull/34/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80